ci/auto/local-mongo44.env

```diff
@@ -1,6 +1,14 @@
-TYK_DB_MONGOURL=mongodb://mongo:27017/tyk_analytics
 TYK_PMP_PUMPS_MONGO_META_MONGOURL=mongodb://mongo:27017/tyk_analytics
 TYK_PMP_PUMPS_MONGO_META_COLLECTIONNAME=tyk_analytics
 TYK_PMP_PUMPS_MONGOAGGREGATE_META_MONGOURL=mongodb://mongo:27017/tyk_analytics
 TYK_PMP_UPTIMEPUMPCONFIG_MONGOURL=mongodb://mongo:27017/tyk_analytics
 TYK_PMP_UPTIMEPUMPCONFIG_COLLECTIONNAME=tyk_analytics
+TYK_PMP_ANALYTICSSTORAGECONFIG_HOST=redis
+TYK_PMP_LOGLEVEL=debug
+TYK_DB_MONGOURL=mongodb://mongo:27017/tyk_analytics
+TYK_DB_LOGLEVEL=debug
+TYK_MDCB_LOGLEVEL=debug
+TYK_MDCB_STORAGE_HOST=redis
+TYK_LOGLEVEL=debug
+TYK_GW_SLAVEOPTIONS_RPCKEY=placeholder
+TYK_GW_SLAVEOPTIONS_APIKEY=placeholder
```

Comment:
This variables shouldn't be here, we need to find another place that makes this sustainable.